.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,69 @@
+name: Bug Report
+description: Something with kwil-db is not working as expected.
```

Comment:
Maybe it was undetected on our other repos, but it's being referred as kwil here 😆